.dev/Dockerfile.debug

```diff
@@ -1,8 +1,7 @@
 FROM node:20.11.0
 WORKDIR /usr/app
-RUN npm install nodemon -g
 
 # Webserver, Debugger
 EXPOSE 80 9229
 
-ENTRYPOINT ["nodemon", "-e", "js,json,yml", "--watch", "./dist", "--watch", "./config", "--inspect=0.0.0.0:9229", "--nolazy", "./dist/index.js"]
+ENTRYPOINT ["npx", "nodemon", "-e", "js,json,yml", "--watch", "./dist", "--watch", "./config", "--inspect=0.0.0.0:9229", "--nolazy", "./dist/index.js"]
```

Comment:
is there a reason for this change?

Reply:
yes this will use the version of nodemon in the node modules that is up to date.
 Without it, it would use a global one that is installed when the image is build and not update afterward

Reply:
great, I think nodemon wasn't available when we created this.